ts/navigation/AuthenticationNavigator.tsx

```diff
@@ -115,15 +114,18 @@ const AuthenticationStackNavigator = () => (
       component={CieConsentDataUsageScreen}
     />
 
-    <Stack.Screen
-      name={ROUTES.CIE_WRONG_PIN_SCREEN}
-      component={CieWrongCiePinScreen}
-    />
-
-    <Stack.Screen
-      name={ROUTES.CIE_PIN_TEMP_LOCKED_SCREEN}
-      component={CiePinLockedTemporarilyScreen}
-    />
+    <Stack.Group
+      screenOptions={{
+        gestureEnabled: false,
+        headerShown: false,
+        ...TransitionPresets.ModalSlideFromBottomIOS
+      }}
+    >
+      <Stack.Screen
+        name={ROUTES.CIE_WRONG_PIN_SCREEN}
+        component={CieWrongCiePinScreen}
+      />
+    </Stack.Group>
```

Comment:
it is not necessary to use the `Stack.Group`, [in the rest of the app](https://github.com/pagopa/io-app/blob/c9ddc8c3914fde18dd2214793a5056f7f0371fa9/ts/navigation/AuthenticatedStackNavigator.tsx#L216) we are using stack.screen, it seems to me that only you used `Stack.Group` (there is another occurrence in the same file)
```suggestion
      <Stack.Screen
        name={ROUTES.CIE_WRONG_PIN_SCREEN}
        component={CieWrongCiePinScreen}
        options={{
          ...TransitionPresets.ModalSlideFromBottomIOS,
          gestureEnabled: false,
          headerShown: false
        }}
      />
```

Reply:
I've used a `Stack.Group` because if we have other EIC screens in the future that needs to be presented as modal, we could put them here. I used [this PR](https://github.com/pagopa/io-app/pull/5577) as reference.

Reply:
This PR from which you have taken your inspiration has not been merged, but has been closed. 
I understand what you say about the stack.group but what is done with the stack.group can be done with the stack.screen, and in this case you only have one screen, so in my opinion it would be more correct to use the stack.screen, any other screens can be grouped in the future or can be used individually with the stack.screen

Reply:
I got what you say, but I prefer to leave it as it is. We have close the other PR for other reasons, not this one. If I remove the `Stack.Group` then I have to reintroduce it on the other PR for the screen I linked above. I don't want to set the modal properties for every screen, but only in a single place.